application/backend/app/services/training/otx_trainer.py

```diff
@@ -127,19 +149,28 @@ def create_training_dataset(self, project_id: UUID, task: TaskBase) -> DatasetIn
                 revision_id=self._dataset_service.save_revision(project_id, dm_dataset),
             )
 
-    @step("Prepare Model Metadata")
+    @step("Prepare Model Metadata & OTX Configuration")
```

Comment:
```suggestion
    @step("Prepare Model and Training Configuration")
```
Slightly more user-friendly, assuming the step name will be displayed in the UI